index.html

```diff
@@ -241,5 +241,41 @@ <h3 class="about-me-item">Skills</h3>
                 </li>
             </ul>
         </section>
+        
+        <footer class="contact-us-footer">
+            <h3 class="contact-us-text">
+                I'm always interested in hearing about 
+                new projects, so if you'd like to chat 
+                please get in touch.
+            </h3>
+
+            <form class="contact-us-form" action="https://formspree.io/f/xknyrjzn" method="POST">
+                <input name="full_name" class="contact-us-input" type="text" placeholder="Full name" maxlength="30" required/>
+                <input name="email" class="contact-us-input" type="email" placeholder="Email" maxlength="30" required/>
+                <textarea name="message" class="contact-us-textarea" placeholder="Enter Text here" maxlength="500"></textarea>
```

Comment:
- [ ] I think you should use `required` attribute for the message input also. :+1: